COMPILING_FROM_SOURCE.md

```diff
@@ -32,12 +32,18 @@ Samples projects are not included in root gradle project, as it is not the way i
 you have to start `build` task from `samples/games/kotlin` directory.
 - Build them manually by directly calling the `build` task of the samples from the IDE or through the commandline:  
 `cd samples/games/kotlin`  
-`./gradlew build -Pplatform=desired_platform`, `desired_platform` can be either `windows`, `linux`, `macos`, `android`.  
-For android, you have to add `android_arch` parameter like:  
+`./gradlew build -Pplatform=desired_platform`, `desired_platform` can be either `windows`, `linux`, `macos`, `android`, `ios`.  
+For android and ios, you have to add `armArch` parameter like:  
 `./gradlew build -Pplatform=android -Pplatform=X64`, otherwise it will build `arm64` platform by default. Supported target
 are for now `arm64` and `X64`  
 or  
 `cd samples/coroutines/kotlin`
-`./gradlew :samples:coroutines:kotlin:build`  
+`./gradlew kotlin:build`  
 Coroutines are not supported on android for now.
  
+Also on ios you should add you signing identity, as it is apple requirement, like:
```

Comment:
I would put this under a separate header. Like "Signing" or something like this. At the moment i gets easily over read.

Reply:
I added a subtitle to make it more clear.